plugins/OneDark/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "author": "Se7en-dev",
+    "name": "OneDark",
+    "description": "One Dark theme for Bridge !",
+    "link": "https://github.com/Se7en-dev/Bridge-OneDark",
+    "version": "1.0.0",
+    "target": "v2",
+    "id": "67ded4a3-f97f-40b3-93b2-86181e15c88d",
+    "icon": "mdi-atom",
+    "tags": [

```

Comment:
The tags list should be tags from [here](https://github.com/bridge-core/editor-packages/blob/main/packages/common/extensionTags.json)
In your case it should just be "Theme"

Reply:
Oops did not see that ! my bad